README.md

```diff
@@ -12,6 +12,10 @@ Hyperopt is a Python library for serial and parallel optimization over awkward
 search spaces, which may include real-valued, discrete, and conditional
 dimensions.
 
+## Requirements
+
+python 3.6+
```

Comment:
@marctorrellas I missed the part about req. 3.6+. As you know I'm willing to drop Py 2 soon, but I'm not convinced it's wise to exclude 3.4 etc.

Good to see that applying this wouldn't be too much work, all in all

Reply:
to be honest, I didn't know either, but black requires 3.6. what we can do as a preliminary step is just to blacken the codebase, without changing the requirements or hooks. This will at least clean the actual state and hopefully it will be less effort in the future

Reply:
what do you think @maxpumperla ? otherwise I'll close this

Reply:
I find this too restrictive for now. let's come back to this once we're ready with dropping python2 support. one step at a time :D

Reply:
oh, forgot to remove this in README. Now it's just black applied, no constraints on Python or hooks